### DISCUSSION TITLE
More Placeholders

### DISCUSSION THREAD

**Discussion by ghost** [Zrips/CMI#4045] [Feature Requests]
Please add more Toplist Placeholders like Playtime, Useranks and more.

**Comment by CloudeLecaw** [comment]
ok so our request is 2 placeholders...
Mire is not a placeholder *g*
You can use a lot of the placeholder from CMI and much much if your using PlaceholderAPI together with the extansions
then you can see playtime allready..

> %cmi_user_charges_left%
> %cmi_user_charges_max%
> %cmi_user_charges_time%
> %cmi_user_charges_cooldown%
> %cmi_user_display_name%
> %cmi_user_nickname%
> %cmi_user_name%
> %cmi_user_deathloc%
> %cmi_user_cuffed%
> %cmi_user_muted%
> %cmi_user_inpvp%
> %cmi_user_god%
> %cmi_user_afk%
> %cmi_user_afk_symbol%
> %cmi_user_afk_msg%
> %cmi_user_afk_for%
> %cmi_user_afk_in%
> %cmi_user_joinedcounter%
> %cmi_user_banned%
> %cmi_user_maxhomes%
> %cmi_user_ping%
> %cmi_user_gamemode%
> %cmi_user_op%
> %cmi_user_canfly%
> %cmi_user_flying%
> %cmi_user_homeamount%
> %cmi_user_vanished_symbol%
> %cmi_user_balance_formated%
> %cmi_user_balance_formatted%
> %cmi_user_balance%
> %cmi_user_prefix%
> %cmi_user_suffix%
> %cmi_user_nameplate_prefix%
> %cmi_user_nameplate_suffix%
> %cmi_user_group%
> %cmi_user_tfly%
> %cmi_user_tfly_formated%
> %cmi_user_tfly_formatted%
> %cmi_user_flightcharge%
> %cmi_user_tgod%
> %cmi_user_tgod_formated%
> %cmi_user_tgod_formatted%
> %cmi_user_votecount%
> %cmi_user_rank%
> %cmi_user_nextranks%
> %cmi_user_nextrankpercent%
> %cmi_user_nextvalidranks%
> %cmi_user_canrankup%
> %cmi_user_country%
> %cmi_user_country_code%
> %cmi_user_city%
> %cmi_user_name_colorcode%
> %cmi_user_glow_code%
> %cmi_user_glow_name%
> %cmi_user_jailname%
> %cmi_user_jailcell%
> %cmi_user_jailtime%
> %cmi_user_jailreason%
> %cmi_user_bungeeserver%
> %cmi_user_playtime_formatted%
> %cmi_user_playtime_days%
> %cmi_user_playtime_dayst%
> %cmi_user_playtime_hours%
> %cmi_user_playtime_hoursf%
> %cmi_user_playtime_hourst%
> %cmi_user_playtime_minutes%
> %cmi_user_playtime_minutest%
> %cmi_user_playtime_seconds%
> %cmi_user_playtime_secondst%
> %cmi_user_itemcount_[itemIdName(:data)]%
> %cmi_user_maxperm_[corePerm]_[defaultValue]%
> %cmi_equation_[equation]%
> %cmi_equationint_[equation]%
> %cmi_iteminhand_displayname%
> %cmi_iteminhand_realname%
> %cmi_iteminhand_type%
> %cmi_iteminhand_itemdata%
> %cmi_iteminhand_amount%
> %cmi_iteminhand_worth%
> %cmi_iteminhand_worth_one%
> %cmi_schedule_nextin_[schedName]%
> %cmi_baltop_name_[1-10]%
> %cmi_baltop_money_[1-10]%
> %cmi_votetop_[1-10]%
> %cmi_votetopcount_[1-10]%
> %cmi_worth_buy_[itemIdName(:data)]%
> %cmi_worth_sell_[itemIdName(:data)]%
> %cmi_bungee_total_[serverName]%
> %cmi_bungee_current_[serverName]%
> %cmi_bungee_motd_[serverName]%
> %cmi_bungee_onlinestatus_[serverName]%
> %cmi_tps_1%
> %cmi_tps_60%
> %cmi_tps_300%
> %cmi_tps_[range]_colored%
> %cmi_random_player_name%
> %cmi_random_[from]_[to]%
> %cmi_lastrandom_[playerName]%
> %cmi_user_rank_percent_[rankName]%
> %cmi_user_meta_[key]%
> %cmi_user_metaint_[key]%
> %cmi_user_kitcd_[kitName]%
> %cmi_user_kit_available_[kitName]%
> %cmi_user_kit_hasaccess_[kitName]%
> %cmi_jail_time_[jailName]_[cellId]%
> %cmi_jail_username_[jailName]_[cellId]%
> %cmi_jail_reason_[jailName]_[cellId]%
> %player_world%
> %player_x%
> %player_y%
> %player_z%
> %player_biome%
> %vault_eco_balance_formatted%
> %server_online%
> %server_max_players%
> %server_unique_joins%
> %onlineplayers_names%
> %onlineplayers_displaynames%
> %server_time_[timeFormat]_[timeZone]%
> %server_time_[timeFormat]%
> %world_time12_[worldName]%
> %world_time24_[worldName]%

and with PlaceholderAPI u extand this to all you find here.
https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Placeholders

so what you really need more?^^
